src/components/InstanceRichTooltipMessage.tsx

```diff
@@ -0,0 +1,122 @@
+import { type FC } from "react";
+import InstanceLink from "pages/instances/InstanceLink";
+import InstanceStatusIcon from "pages/instances/InstanceStatusIcon";
+import { instanceCreationTypes } from "util/instanceOptions";
+import { useInstance } from "context/useInstances";
+import { Spinner } from "@canonical/react-components";
+import { getIpAddresses, sortIpv6Addresses } from "util/networks";
+import type { IpFamily, LxdInstance } from "types/instance";
+import InstanceListAddresses from "pages/instances/InstanceListAddresses";
+import { getInstanceMacAddresses } from "util/instances";
+import {
+  useIsScreenBelow,
+  largeScreenBreakpoint,
+} from "context/useIsScreenBelow";
+import { isoTimeToString } from "util/helpers";
+
+interface Props {
+  instanceName: string;
+  projectName: string;
+}
+
+export const InstanceRichTooltipMessage: FC<Props> = ({
+  instanceName,
+  projectName,
+}) => {
+  const getAddresses = (instance: LxdInstance, family: IpFamily) => {
+    const addresses = getIpAddresses(instance, family);
+    const sortedAddresses =
+      family === "inet6" ? sortIpv6Addresses(addresses) : addresses;
+    return sortedAddresses.map((item) => item.address);
+  };
+
+  const { data: instance, isLoading } = useInstance(instanceName, projectName);
+  const isLargeScreen = !useIsScreenBelow(largeScreenBreakpoint);
+
+  if (!instance && !isLoading) {
+    return <></>;
+  }
+  const ipv4Addresses = instance ? getAddresses(instance, "inet") : [];
+  const ipv6Addresses = instance ? getAddresses(instance, "inet6") : [];
+  const macAddresses = instance ? getInstanceMacAddresses(instance) : [];
+
+  return (
+    <table className="u-table-layout--auto u-no-margin--bottom instance-rich-tooltip-table">
+      <tbody>
+        <tr>
+          <th className="u-text--muted">Name</th>
+          <td title={instance ? instance.name : "-"} className="u-truncate">
```

Comment:
```suggestion
          <td title={instance?.name} className="u-truncate">
```
I think when there is no content, we don't need the title. Its main use is for the overflow case for when the content is truncated and only half visible.